### ISSUE TITLE
The resource attribute contains "/core-model-1-4:control-construct" twice.

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/NotificationProxy#74]
The resource attribute contains "/core-model-1-4:control-construct" twice.
```
"resource":"/core-model-1-4:network-control-domain=live/control-construct=xxx/core-model-1-4:control-construct/equipment=xxx",

```
{"notification-proxy-1-0:alarm-event-notification":{
"alarm-event-sequence-number":1702,
"timestamp":"2024-02-14T05:53:11+00:00",
"resource":"/core-model-1-4:network-control-domain=live/control-construct=CO17694/core-model-1-4:control-construct/equipment=HUAWEI-EQUIPMENT-1",
"alarm-type-id":"alarms-1-0:ALARM_TYPE_ID_TYPE_NE_OBJ",
"alarm-type-qualifier":"DCNSIZE_OVER/13110-20/FFFFFFFFFF",
"problem-severity":"major"
}
}

**Comment by paulBMentopolis** [comment]
Fixed in tag "v1.0.1_impl-fix-3"

**Comment by Ziabatcha** [comment]
Verified in "v1.0.1_impl-fix-3" and issue not observed hence closing it 
logs:
 {"notification-proxy-1-0:alarm-event-notification":{"alarm-event-sequence-number":373,"timestamp":"2024-03-05T08:58:49+00:00","resource":"/core-model-1-4:network-control-domain=live/control-construct=CO17695/equipment=HUAWEI-EQUIPMENT-1","alarm-type-id":"alarms-1-0:ALARM_TYPE_ID_TYPE_NE_OBJ","alarm-type-qualifier":"DCNSIZE_OVER/13110-20/FFFFFFFFFF","problem-severity":"major"}}
